pulp_rpm/app/fields.py

```diff
@@ -39,6 +40,8 @@ def to_representation(self, value):
                     'sum_type': pkg['sum_type'],
                     'version': pkg['version'],
                 })
+            if collection.module != '':
```

Comment:
Can `collection.module` be `None`?
Maybe it's safer to do `if not collection.module`. What do you think?

Reply:
this was a first try, but as no module present an empty string is returned (so with none it is always true)